### ISSUE TITLE
Problem with Timmy

### ISSUE THREAD

**Issue by fatk** [timber/timber#2276]
After a Timber update, Timmy Twig filters stopped being registered.

## Expected behavior
Filter ``timber/twig`` should fire to trigger Timmy to setup its own Twig filters

## Actual behavior
Filter ``timber/twig`` doesn't seem to fire anymore

## Steps to reproduce behavior
Update Timber from ``1.14.0`` to ``1.16.0`` (maybe ``1.15.0`` is  also affected)

## What version of WordPress, PHP and Timber are you using?
WordPress 5.4.2, Timber 1.16.0

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Composer

## Important
I'm using the Themosis Framework, and i'm loading Timber through a Laravel ServiceProvider:

```
<?php

namespace Theme\Services;

use Timber\Timber;
use Timmy\Timmy;

class TimberService
{
    protected $timber;
    protected $timmy;
    protected $twig;

    public function __construct($container)
    {
        $this->timber = new Timber();
        $this->timmy = new Timmy();
        $this->twig = $container['twig'];
    }

    public function install()
    {
        apply_filters('twig_apply_filters', $this->twig);
        apply_filters('timber/twig/filters', $this->twig);
        apply_filters('timber/twig/functions', $this->twig);
        apply_filters('timber/twig/escapers', $this->twig);
        apply_filters('timber/loader/twig', $this->twig);
    }
}
```

**Comment by gchtr** [comment]
This might very well be related to the changes in https://github.com/timber/timber/pull/2238.

I’m not familiar with Themosis or how to integrate Timber with a Laravel ServiceProvider. Butn when I look at the following lines and your calls to `apply_filters`, then probably `timber/twig` isn’t applied in your case:

https://github.com/timber/timber/blob/5b23f146e510a35b2deae645fc63d1bc8a4e3545/lib/Loader.php#L184-L193

Can you quickly check whether

- using `apply_filters('timber/loader/twig', $this->twig);` instead of `apply_filters('timber/twig', $this->twig);` in `TimberService::install()` or
- moving the line `$twig = apply_filters('timber/loader/twig', $twig);` below the line that reads `$twig = apply_filters('timber/twig', $twig);` in [lib/Loader.php](https://github.com/timber/timber/blob/5b23f146e510a35b2deae645fc63d1bc8a4e3545/lib/Loader.php#L184)

solves the issue?

**Comment by fatk** [comment]
Thanks for the quick reply and interest!

I've just updated to 1.16 again, and as soon as i edited lib/Loader.php, the problem disappeared.

Turns out the problem was with Docker's Hypervisor filesystem cache implementation when updating file through a dockerized version of composer.

I didn't think to go down this path because Timber updates never gave me any problem before, but it looks like the changes in #2238 were not completely applied and caused the issue.

Thank you for your time and help, I've been using Timber for over 4 years now i think and it's been astoundingly helpful.
